Sources/TXCli/main.swift

```diff
@@ -234,7 +244,7 @@ or via the --token parameter.
 
     @Option(name: .long, parsing: .upToNextOption, help: """
 A list of the available locales that the application supports and will be
-downloaded from CDS. The source locale, if added, is ignored.
+downloaded from CDS.
```

Comment:
Maybe it's even better if we explicitly note that both source and target locales are included.

Reply:
Done. I have amended my original commit. Let me know what you think.